server/models/Request.js

```diff
@@ -0,0 +1,32 @@
+const mongoose = require("mongoose");
+
+const requestSchema = new mongoose.Schema({
+  ownerId: {
+    type: mongoose.ObjectId,
+    required: true,
+    ref: 'User'
+  },
+  sitterId: {
+    type: mongoose.ObjectId,
```

Comment:
This should be `Schema.Types.ObjectId`, while `Schema` is a property on `mongoose`